packages/react-lab/src/date-picker/Calendar/Calendar.js

```diff
@@ -0,0 +1,104 @@
+import { Box } from '@tonic-ui/react';
+import { usePrevious } from '@tonic-ui/react-hooks';
+import isSameMonth from 'date-fns/isSameMonth';
+import isSameYear from 'date-fns/isSameYear';
+import React, { forwardRef, useEffect, useState } from 'react';
+import DecadeView from './DecadeView';
+import MonthView from './MonthView';
+import YearView from './YearView';
+import Navigation from './Navigation';
+import { useCalendarStyle } from './styles';
+import {
+  isDate,
+  toDate,
+} from '../utils';
+
+const Calendar = forwardRef((
+  {
+    calendarStartDay = 0, // 0 = Sunday, 1 = Monday, ...
+    children,
+    dateFormat = 'yyyy-MM-dd',
+    value = new Date(),
```

Comment:
Add `defaultValue` prop to make it uncontrolled.

Reply:
You can change `value = new Date()` to `value` . Only the `defaultValue` prop may use current date as its default value.

See https://github.com/trendmicro-frontend/tonic-ui/blob/master/packages/react/src/accordion/AccordionItem.js for reference.